semcore/base-trigger/src/BaseTrigger.jsx

```diff
@@ -56,7 +56,13 @@ class RootBaseTrigger extends Component {
       <NeighborLocation.Detect neighborLocation={neighborLocation}>
         {(neighborLocation) =>
           sstyled(styles)(
-            <SBaseTrigger render={Box} neighborLocation={neighborLocation} state={theme}>
+            <SBaseTrigger
+              render={Box}
+              tag='button'
```

Comment:
btw, shouldn't we remove tag='button' and type='button' from descendants, like button trigger and filter trigger? 
to reduce redundancy

Reply:
I agree with Sheila. Let's remove it there

Reply:
Removed from Button and Filter triggers, but kept for LinkTrigger as LinkTrigger is not descendant of BaseTrigger